packages/dbml-core/__tests__/parser/mssql-parse/input/inline_referential_actions.in.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE [orders] (
   [id] int PRIMARY KEY IDENTITY(1, 1),
   [user_id] int UNIQUE NOT NULL 
-    CONSTRAINT fk_order_user FOREIGN KEY REFERENCES [users] ([id]) 
+    CONSTRAINT fk_order_user FOREIGN KEY REFERENCES [users] 
```

Comment:
Can u explain why u need to change these test cases?

Reply:
For testing implicit fk reference feature that I also implement in this branch